libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,563 @@
+    method public void setRouteLineOptions(com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? p);
+    property public final String? mapStyleUriDay;
+    property public final String? mapStyleUriNight;
+    property public final com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? routeArrowOptions;
+    property public final com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? routeLineOptions;
+    field public static final com.mapbox.navigation.dropin.ViewOptionsCustomization.Companion Companion;
+  }
+
+  public static final class ViewOptionsCustomization.Companion {
+    method public com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions defaultRouteArrowOptions(android.content.Context context);
+    method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions defaultRouteLineOptions(android.content.Context context);
+  }
+
+}
+
+package com.mapbox.navigation.dropin.binder {
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public interface Binder<T> {
+    method public com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver bind(T? viewGroup);
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public fun interface UIBinder extends com.mapbox.navigation.dropin.binder.Binder<android.view.ViewGroup> {
+    field public static final com.mapbox.navigation.dropin.binder.UIBinder.Companion Companion;
+  }
+
+  public static final class UIBinder.Companion {
+    method public com.mapbox.navigation.dropin.binder.UIBinder getUSE_DEFAULT();
+    property public final com.mapbox.navigation.dropin.binder.UIBinder USE_DEFAULT;
+  }
+
+  public final class UIBinderKt {
+  }
+
+}
+
+package com.mapbox.navigation.dropin.component.audioguidance {
+
+  public abstract sealed class AudioAction {
+  }
+
+  public static final class AudioAction.Mute extends com.mapbox.navigation.dropin.component.audioguidance.AudioAction {
+    field public static final com.mapbox.navigation.dropin.component.audioguidance.AudioAction.Mute INSTANCE;
+  }
+
+  public static final class AudioAction.Toggle extends com.mapbox.navigation.dropin.component.audioguidance.AudioAction {
+    field public static final com.mapbox.navigation.dropin.component.audioguidance.AudioAction.Toggle INSTANCE;
+  }
+
+  public static final class AudioAction.Unmute extends com.mapbox.navigation.dropin.component.audioguidance.AudioAction {
+    field public static final com.mapbox.navigation.dropin.component.audioguidance.AudioAction.Unmute INSTANCE;
+  }
+
+  public final class AudioGuidanceState {
+    method public boolean component1();
+    method public com.mapbox.navigation.dropin.component.audioguidance.AudioGuidanceState copy(boolean isMuted);
+    method public boolean isMuted();
+    property public final boolean isMuted;
+  }
+
+}
+
+package com.mapbox.navigation.dropin.component.camera {
+
+  public abstract sealed class CameraAction {
```

Comment:
Can this be converted to internal?

Reply:
Same as https://github.com/mapbox/mapbox-navigation-android/pull/5703#discussion_r849945755. Resolving so we can track it in one place.